src/types/sequence.rs

```diff
@@ -338,11 +333,15 @@ impl<'v> PyTryFrom<'v> for PySequence {
     fn try_from<V: Into<&'v PyAny>>(value: V) -> Result<&'v PySequence, PyDowncastError<'v>> {
         let value = value.into();
```

Comment:
I think this ties in somewhat with the discussion to generally overhaul the conversion traits, but if the `Into::into` call is the only reason this method is generic, wouldn't an inner function taking a plain `&PyAny` be a reasonable measure against code bloat?

Reply:
I agree and actually in my #1308 proposal I hope to push soon I want to just make the whole method take `&PyAny`.

Reply:
Fair enough to leave it like this as-is until we resolve the underlying issue.